esmvaltool/config-references.yml

```
@@ -414,9 +414,9 @@ authors:
     institute: University of Bremen and DLR, Germany
     orcid: https://orcid.org/0000-0001-6133-7801
   wenzel_sabrina:
```

```suggestion
  zechlau_sabrina:
```
and also place it in the correct alphabetical order.